src/new-apps/components/AppAvatar/AppAvatar.tsx

```diff
@@ -0,0 +1,46 @@
+import genericAppImg from "@assets/images/app-generic.svg";
+import { AppLogo } from "@dashboard/new-apps/types";
+import { Box } from "@saleor/macaw-ui/next";
+import React from "react";
+
+interface AppAvatarProps {
+  logo: AppLogo | undefined;
+  size: "medium" | "large";
+}
+
+export const AppAvatar = (props: AppAvatarProps) => {
+  const { logo, size } = props;
+
+  const avatarSize = size === "medium" ? 11 : 13;
+
+  if (logo?.source) {
+    return (
+      <Box
+        __backgroundColor={logo?.color}
+        padding={3}
+        width={avatarSize}
+        height={avatarSize}
+        borderRadius={2}
+        display="flex"
+        placeItems="center"
+      >
+        <Box as="img" src={logo.source} />
+      </Box>
+    );
+  } else {
+    return (
+      <Box
+        __backgroundColor={logo?.color ?? "#EAE8E9"}
```

Comment:
question: can't we take this color from theme vars?

Reply:
No, because these colors are not available in the new implementation of macaw-ui. We will remove them when UI is merged to new colors.